etna/transforms/detrend.py

```diff
@@ -44,6 +44,7 @@ def fit(self, df: pd.DataFrame) -> "_OneSegmentLinearTrendBaseTransform":
         _OneSegmentLinearTrendBaseTransform
             instance with trained regressor
         """
+        df = df.dropna()
```

Comment:
What if `None` in the middle of df?
We could use something like this https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.first_valid_index.html